### PR TITLE
Improved conversation list design and updated launcher icon

### PR DESCRIPTION
**Pull request by AndyScherzinger**
Basically backporting conversation list changes (UI only) and launcher icon from `master-broken`-PR #1062 to `master`.

|:sunny:light (old)|:sun_with_face:light (new)|:new_moon:dark (old)|:new_moon_with_face:dark (new)|
|---|---|---|---|
|light|light_new|dark|dark_new|

Things that changed (as per material design guideline)
* background: light is `#ffffff` and dark is `#121212`
* high empahsis text (as in headlines) are `#deffffff` as in `#ffffff` at opacity level 87%
* distance/whitespace between headline and message text has been slightly lowered
* unread message bubble is now a chip view (as in pill-style)
* conversation with unread messages uses a bold text style to be easier to spot
* space between avatar and text has been raised

Other than that the launcher icon is now aligned files the files/deck/notes etc. Android clients
ic_launcher

@mahibi for bonus points I also fixed the lint result Github comment part since the master values need to be taked from master-branch not the result file of the actual branch :) - basically copied @tobiasKaminsky's code from the files-client :)

### CODE REVIEW

File: app/src/main/java/com/nextcloud/talk/adapters/items/ConversationItem.java [hunk 1]
```diff
@@ -113,17 +118,32 @@ public void bindViewHolder(FlexibleAdapter<IFlexible> adapter, ConversationItemV
...
+                    holder.dialogDate.getTypeface(),
+                    Typeface.BOLD
+            );
+            holder.dialogLastMessage.setTypeface(
+                    holder.dialogLastMessage.getTypeface(),
+                    Typeface.BOLD
+            );
             holder.dialogUnreadBubble.setVisibility(View.VISIBLE);
-            if (conversation.getUnreadMessages() < 100) {
+            if (conversation.getUnreadMessages() < 1000) {
```
Comment (nickvergessen) [inline app/src/main/java/com/nextcloud/talk/adapters/items/ConversationItem.java:134]: Finally 😎

File: app/src/main/res/drawable-v24/ic_launcher_background.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<!--
+  ~ /*
+  ~  * Nextcloud Talk application
+  ~  *
+  ~  * @author Mario Danic
+  ~  * Copyright (C) 2017-2020 Mario Danic <mario@lovelyhq.com>
```
Comment (nickvergessen) [inline app/src/main/res/drawable-v24/ic_launcher_background.xml:6]: 😕

File: app/src/main/res/drawable-v24/ic_launcher_background.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  ~  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~  * GNU General Public License for more details.
+  ~  *
+  ~  * You should have received a copy of the GNU General Public License
+  ~  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  ~  */
+  -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (nickvergessen) [inline app/src/main/res/drawable-v24/ic_launcher_background.xml:23]: Finally :)

File: app/src/main/res/values-night/colors.xml [hunk 4]
```diff
@@ -22,16 +24,19 @@
   -->
 
 <resources>
-    <color name="conversation_item_header">#ffffff</color>
+    <color name="conversation_item_header">#deffffff</color>
 
-    <color name="bg_default">#222222</color>
-    <color name="bg_alt">#222222</color>
+    <color name="bg_default">#121212</color>
```
Comment (nickvergessen) [inline app/src/main/res/values-night/colors.xml:29]: 😎

File: app/src/main/res/values/colors.xml [hunk 5]
```diff
@@ -59,5 +62,5 @@
     <color name="bg_bottom_sheet">#46ffffff</color>
 
     <color name="call_buttons_background">#BF999999</color>
-
+    <color name="favorite_icon_tint">#FFCC00</color>
```
Comment (nickvergessen) [inline app/src/main/res/values/colors.xml:65]: Finally ⭐

File: app/src/main/java/com/nextcloud/talk/adapters/items/ConversationItem.java (hunk 1 above)
Comment (AndyScherzinger) [reply app/src/main/java/com/nextcloud/talk/adapters/items/ConversationItem.java:134]: My pleasure 😄

File: app/src/main/res/drawable-v24/ic_launcher_background.xml (hunk 2 above)
Comment (AndyScherzinger) [reply app/src/main/res/drawable-v24/ic_launcher_background.xml:6]: Backported and it is in the original file, so... left it in...

File: app/src/main/res/values/colors.xml (hunk 5 above)
Comment (AndyScherzinger) [reply app/src/main/res/values/colors.xml:65]: Jupp :)

Comment (AndyScherzinger) [reply app/src/main/res/values/colors.xml:65]: Reason I did all this: Message loading and posting is broken on `master-broken` where the UI is nice, master works but UI needs polishing. So backport changes to have a working _and_ nice client on Android :)

**Review by ma12-co — APPROVED** [review]
Very nice improvements :)

**Comment by mahibi** [comment]
@AndyScherzinger 👌👌👌

**Comment by AndyScherzinger** [comment]
Manual Backport: #1089

**Pull request merged**